functional-tests/features/404.feature

```diff
@@ -0,0 +1,17 @@
+Feature: 404 Page
+  As a user of CKS
+  I want to see the CKS 404 Page when a page is not found
+
+  Background:
+    Given I open the url "/random-route"
+
+  Scenario: User can see the 404 page title
```

Comment:
Maybe a name like:
```suggestion
  Scenario: User sees custom content for 404 response
```
Cos the behaviour we're testing is that we see the custom content - we're not fussed that specifically it's the title

Reply:
done